sdk/python/jobs/finetuning/standalone/chat-completion/chat_completion_with_model_as_service.ipynb

```diff
@@ -0,0 +1,559 @@
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities._inputs_outputs import Input\n",
+    "\n",
+    "training_data = Input(\n",
+    "    type=train_data_created.type,\n",
+    "    path=f\"azureml://locations/{workspace.location}/workspaces/{workspace._workspace_id}/data/{train_data_created.name}/versions/{train_data_created.version}\",\n",
+    ")\n",
+    "validation_data = Input(\n",
+    "    type=validation_data_created.type,\n",
+    "    path=f\"azureml://locations/{workspace.location}/workspaces/{workspace._workspace_id}/data/{validation_data_created.name}/versions/{validation_data_created.version}\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Submit the fine tuning job using the the model and data as inputs\n",
+    " \n",
+    "Create FineTuning job using all the data that we have so far."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Create marketplace subscription for 3P models\n",
+    "Note: Skip this step for 1P models that are offered on Azure. Example: Phi family of models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_id_to_subscribe = \"/\".join(foundation_model.id.split(\"/\")[:-2])\n",
+    "print(model_id_to_subscribe)\n",
+    "\n",
+    "normalized_model_name = model_name.replace(\".\", \"-\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import MarketplaceSubscription\n",
+    "\n",
+    "\n",
+    "subscription_name = f\"{normalized_model_name}-sub\"\n",
+    "\n",
+    "marketplace_subscription = MarketplaceSubscription(\n",
+    "    model_id=model_id_to_subscribe,\n",
+    "    name=subscription_name,\n",
+    ")\n",
+    "\n",
+    "# note: this will throw exception if the subscription already exists or subscription is not required (for example, if the model is not in the marketplace like Phi family)\n",
+    "try:\n",
+    "    marketplace_subscription = (\n",
+    "        workspace_ml_client.marketplace_subscriptions.begin_create_or_update(\n",
+    "            marketplace_subscription\n",
+    "        ).result()\n",
+    "    )\n",
+    "except Exception as ex:\n",
+    "    print(ex)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define finetune parameters\n",
+    "\n",
+    "##### There are following set of parameters that are required.\n",
+    "\n",
+    "1. `model` - Base model to finetune.\n",
+    "2. `training_data` - Training data for finetuning the base model.\n",
+    "3. `task` - FineTuning task to perform. eg. TEXT_COMPLETION for text-generation/text-generation finetuning jobs.\n",
+    "4. `outputs`- Output registered model name.\n",
+    "\n",
+    "##### Following parameters are optional:\n",
+    "\n",
```

Comment:
validation_data needs to be added as well since an asset is created and used in the job